services/app/src/pages/game/Game.jsx

```diff
@@ -53,12 +55,25 @@ function Room() {
     [users, currentUser, movedBy],
   );
 
+  useEffect(() => {
+    if (playerDeleted && playerDeleted === currentUser.id) {
```

Comment:
I've tried to open two different browser windows and tried to remove one of the players, but it does not seem to work, as the removed player is not being redirected to login screen after removal.